starter/package.json

```diff
@@ -30,8 +31,9 @@
     "@radix-ui/react-tooltip": "^1.1.2",
     "@types/mapbox-gl": "^2.7.5",
     "@types/node": "^20.12.3",
-    "@yext/search-headless-react": "^2.5.2",
-    "@yext/search-ui-react": "^1.8.13",
+    "@yext/search-headless-react": "^2.5.3",
+    "@yext/search-ui-react": "^1.9.0",
+    "@yext/visual-editor": "*",
```

Comment:
`"@yext/visual-editor": "workspace:*",` is already in devDependencies. It's in the wrong place, should be normal deps. But is there a way to make it work with workspace? Idk what happens if both exist.